app/controllers/teams_controller.rb

```diff
@@ -0,0 +1,60 @@
+class TeamsController < ApplicationController
+  include HasHarvest
+
+  def index
+    @teams = Team.active
+  end
+
+  def new
+    @team = Team.new
+    @harvest_projects = harvest_projects_list
+  end
+
+  def create
+    @team = Team.new(team_attributes)
+
+    if @team.save
+      redirect_to teams_path
+    else
+      render :new
```

Comment:
Wouldn't you need `@harvest_projects` set again if you were to re-render `:new`?